desktop/core/src/desktop/urls.py

```diff
@@ -202,6 +203,9 @@
 ]
 
 dynamic_patterns += [
+  re_path('api/schema', SpectacularAPIView.as_view(),name='schema'),
```

Comment:
We can remove this mapping as well, I checked and its just downloading the schema for all the APIs. Is this required?

Reply:
yes, we require this for QE.

Reply:
Okay then we can change the mapping here to '^api/schema/?$'?